roles/setup_http_store/tasks/main.yml

```diff
@@ -46,14 +50,14 @@
         volumes:
           - "{{ http_data_dir }}:/var/www/html:z"
       register: container_info
-  become: true
 
 - name: Setting facts about container
```

Comment:
This seems to be the only task in the file not in a `become: true` block, and there's no reason for it not to be IMO.